src/Magnum/OvrIntegration/Hmd.h

```diff
@@ -358,24 +370,110 @@ class MAGNUM_OVRINTEGRATION_EXPORT Hmd {
         StatusFlags trackingState() const {
             return {StatusFlag(_trackingState.StatusFlags)};
         }
 
+        /** @brief Name of the currently active Oculus profile */
+        std::string user() const {
+            return ovr_GetString(_session, OVR_KEY_USER, "");
+        }
+
+        /** @brief Name set in the currently active Oculus profile */
+        std::string playerName() const {
+            return ovr_GetString(_session, OVR_KEY_NAME, "");
+        }
+
+        /** @brief Gender set in the currently active Oculus profile */
+        std::string playerGender() const {
+            return ovr_GetString(_session, OVR_KEY_GENDER, OVR_DEFAULT_GENDER);
+        }
+
+        /** @brief Player height set in the currently active Oculus profile */
+        Float playerHeight() const {
+            return ovr_GetFloat(_session, OVR_KEY_EYE_HEIGHT, OVR_DEFAULT_PLAYER_HEIGHT);
+        }
+
+        /** @brief Eye height set in the currently active Oculus profile */
+        Float eyeHeight() const {
+            return ovr_GetFloat(_session, OVR_KEY_EYE_HEIGHT, OVR_DEFAULT_EYE_HEIGHT);
+        }
+
+        /**
+         * @brief Inter populary distance set in the currently active Oculus
+         * profile
+         */
+        Float ipd() const {
+            return ovr_GetFloat(_session, OVR_KEY_IPD, OVR_DEFAULT_IPD);
+        }
+
+        /**
+         * @brief Neck to eye distance set in the currently active Oculus
+         * profile
+         */
+        std::array<Float, 2> neckToEyeDistance() const {
+            std::array<Float, 2> values{{OVR_DEFAULT_NECK_TO_EYE_HORIZONTAL, OVR_DEFAULT_NECK_TO_EYE_VERTICAL}};
+            ovr_GetFloatArray(_session, OVR_KEY_NECK_TO_EYE_DISTANCE, values.data(), 2);
+            return values;
+        }
+
+        /**
+         * @brief State of the eye relief dial set in the currently active
+         * Oculus profile
+         */
+        Int eyeReliefDial() const {
+            return ovr_GetInt(_session, OVR_KEY_EYE_RELIEF_DIAL, OVR_DEFAULT_EYE_RELIEF_DIAL);
+        }
+
+        /**
+         * @brief Eye to node distance set in the currently active Oculus
+         * profile
+         */
+        std::array<Float, 2> eyeToNoseDistance() const {
+           std::array<Float, 2> values{{0.0f, 0.0f}};
+            ovr_GetFloatArray(_session, OVR_KEY_EYE_TO_NOSE_DISTANCE, values.data(), 2);
+            return values;
+        }
+
+        /**
+         * @brief Maximal eye to plate distance set in the currently active
+         * Oculus profile
+         */
+        std::array<Float, 2> maxEyeToPlateDistance() const {
+            std::array<Float, 2> values{{0.0f, 0.0f}};
+            ovr_GetFloatArray(_session, OVR_KEY_MAX_EYE_TO_PLATE_DISTANCE, values.data(), 2);
+            return values;
+        }
+
+        /** @brief Eye cup set in the currently active Oculus profile */
+        std::string eyeCup() const {
+            return ovr_GetString(_session, OVR_KEY_EYE_CUP, "A");
```

Comment:
I guess that is the default, and I probably took that from some defaults by the SDK itself, I'll check again, though, just to be on the safe side (in my oppinion "B" would be a more reasonable default, since it's the lense without sight correction)